devices/Paexo/conf/Paexo.xml

```diff
@@ -64,4 +72,41 @@
         <action phase="shutdown" level="4" type="detach"/>
     </device>
 
+    <device type="iwear_logger" name="ProducerLoggerDevice">
+    <param name="period">0.01</param>
+
+    <param name="logAllQuantities">true</param>
+    <param name="logAccelerometers">false</param>
+    <param name="logEMGSensors">false</param>
+    <param name="logForce3DSensors">false</param>
+    <param name="logForceTorque6DSensors">true</param>
+    <param name="logFreeBodyAccelerationSensors">true</param>
+    <param name="logGyroscopes">true</param>
+    <param name="logMagnetometers">false</param>
+    <param name="logOrientationSensors">true</param>
+    <param name="logPoseSensors">false</param>
+    <param name="logPositionSensors">false</param>
+    <param name="logTemperatureSensors">false</param>
+    <param name="logTorque3DSensors">false</param>
+    <param name="logVirtualLinkKinSensors">false</param>
+    <param name="logVirtualJointKinSensors">false</param>
+    <param name="logVirtualSphericalJointKinSensors">false</param>
+
+    <param name="saveBufferManagerConfiguration">true</param>
+    <param name="experimentName">test_iwear_logger</param>
+    <param name="path">/home/yeshi/Desktop/Paexo-test/</param>
+    <param name="n_samples">100000</param>
+    <param name="save_periodically">true</param>
+    <param name="save_period">120.0</param>
+    <param name="data_threshold">300</param>
+    <param name="auto_save">true</param>
+
+    <action phase="startup" level="5" type="attach">
+        <paramlist name="networks">
+            <elem name="ProducerDeviceWrapperLabel"> PaexoWearableDevice </elem>
+        </paramlist>
+    </action>
+    <action phase="shutdown" level="5" type="detach"/>
+</device>
+
```

Comment:
I wouldn't avoid committing the device with  the `iwear_logger` always enabled.
Probably you can leave it commented, and change the `path` parameter with a general path.

Reply:
I was referring to the `iwear_logger` device configuration https://github.com/robotology/wearables/pull/117/files#diff-66c831614f6cc22f28d917123bffd0dbbba579ed8a45ff08841a1887ba93e213R75-R110.

it seems the as `path` configuration you are passing your local path `<param name="path">/home/yeshi/Desktop/Paexo-test/</param>`, while you can leave something generic in the commit, such as `<param name="path">/path/to/some/folder/</param>`.

And the other point was that you could comment out the `iwear_logger` device since as default usage it is not required to dump al the data.

Reply:
Done in https://github.com/robotology/wearables/pull/117/commits/97cf9e256bb5cd4cf1167d39d32f7aa1523bdc52